editor/plugin_config_dialog.cpp

```diff
@@ -73,6 +73,7 @@ void PluginConfigDialog::_on_confirmed() {
 		// TODO Use script templates. Right now, this code won't add the 'tool' annotation to other languages.
 		// TODO Better support script languages with named classes (has_named_classes).
 
+#ifdef GDSCRIPT_ENABLED
```

Comment:
This should eventually be fixed to remove the dependency on GDScript, but that's a topic for another issue/PR.